zerver/lib/integrations.py

```diff
@@ -94,6 +96,40 @@ def add_doc_context(self, context):
         # type: (Dict[Any, Any]) -> None
         self.doc_context = context
 
+class BotIntegration(Integration):
+    DEFAULT_LOGO_STATIC_PATH_PNG = 'static/images/bots/{name}.png'
+    DEFAULT_LOGO_STATIC_PATH_SVG = 'static/images/bots/{name}.svg'
+    DEFAULT_AVATAR_STATIC_PATH_PNG = 'static/images/bots/default-avatar.png'
```

Comment:
Hmm, the default avatar issue poses an interesting dilemma.  We probably want these to live in the repository with the bots, not in the main Zulip repository, right?  I don't think it makes sense to check in a  new file for this; we should either not have one set (so that we'd use a random gravatar) or have it actually be something related to the bot.